### ISSUE TITLE
Create a web app translating from Html to Js

### ISSUE THREAD

**Issue by elroykanye** [osscameroon/js-generator#6]
I would like to help out with this. Am I open to using any tools at my disposal for accomplishing it?

**Comment by FanJups** [comment]
Hi @elroykanye 😊,
Thanks for this issue.
Let's focus on the features first.

**Comment by elroykanye** [comment]
Okay, do you refer to those listed as todos on this repo's README?

**Comment by FanJups** [comment]
Yes, before that we will look your PR. After merging, we will organize a call to talk about the project.

**Comment by elroykanye** [comment]
okay, noted

**Comment by FanJups** [comment]
Some illustrations for the web app: https://hilla.dev/

**Comment by FanJups** [comment]
Some illustrations for the web app: https://www.postman.com/

![image](https://user-images.githubusercontent.com/25267726/211171833-78c639a9-58cf-4ae3-a0d3-2d26e8e6c6b5.png)
